CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/tasks/LookupPluginsTask.java

```diff
@@ -50,7 +56,26 @@ public Map<String, List<DataAccessPlugin>> get() {
                 }
             }
         }
-        return plugins;
+
+        if (StringUtils.isNotBlank(VISIBLE_CATS)) {
+            final String[] visibleCategoriesArray = addCategoryToList(VISIBLE_CATS);
+            if (visibleCategoriesArray.length > 0) {
+                plugins.keySet().forEach(key -> {
+                    for (int i = 0; i < visibleCategoriesArray.length; i++) {
+                        if (key.equals(visibleCategoriesArray[i])) {
+                            final Pair<Integer, List<DataAccessPlugin>> p = new Pair<>(i, plugins.get(key));
+                            pluginsWithOrder.put(key, p);
+                            break;
+                        }
+                    }
+                });
+            }
+        }
+
+        if (StringUtils.isBlank(VISIBLE_CATS) && StringUtils.isBlank(DAV_CATS)) {
```

Comment:
given the if statement immediately above it is `StringUtils.isNotBlank(VISIBLE_CATS)`, would it make more sense to make this an else-if condition of that? It would mean you wouldn't need to include the first boolean statement here (as that will be implied by the if condition falling to the else-if). i.e.
```
if (StringUtils.isNotBlank(VISIBLE_CATS)) {
    ...
} else if (StringUtils.isBlank(DAV_CATS)) {
    ...
}
```